common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptDao.java

```diff
@@ -46,7 +46,7 @@
     @Query(value = "select c.* from concept c join db_domain d on c.concept_id=d.concept_id", nativeQuery = true)
     List<Concept> findDbDomainParticpantCounts();
 
-    @Query(value = "select distinct c.conceptId from Concept c left join c.synonyms as cs where match(cs.conceptSynonymName,?1) > 0")
+    @Query(value = "select distinct c.conceptId from Concept c left join c.synonyms as cs where match(cs.conceptSynonymName,?1) > 0 or match(c.conceptName,?1) > 0")
```

Comment:
This function should be named better after what it does, how about findConceptIdsByNameOrSynonymMatch 
Or if you want more general : findConceptIdsByQuery